### ISSUE TITLE
Table: ColumnFilter type boolean missing ariaLabel

### ISSUE THREAD

**Issue by michelAlexis** [primefaces/primeng#19314]
### Describe the bug

The ColumnFilter component does not forward the property `ariaLabel` to the underline checkbox.

### Pull Request Link

#19315 

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://stackblitz.com/edit/7e1agk6n?file=src%2Fapp%2Ftable-filterbasic-demo.ts

Columne "verified" has no aria-label

### Environment

None

### Angular version

^21.0.0

### PrimeNG version

v21

### Steps to reproduce the behavior

Add a ColumnFilter of type `boolean` and a property `ariaLabel`.
The underline p-checkbox does not have an `arialLabel`

### Expected behavior

The p-checkbox should have the ariaLabel
